azure-group-deploy.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -e
-while getopts "a:l:g:s:f:e:u" opt; do
+while getopts "a:l:g:s:f:e:u:v" opt; do
```

Comment:
Ok, thanks - you mean like this: https://github.com/Azure/azure-quickstart-templates/pull/2650